src/svgSprite.ts

```diff
@@ -542,5 +542,23 @@ export const svgSprite = (): string => `
       </clipPath>
     </defs>
   </symbol>
+
+  <symbol viewBox="0 0 19 18" id="svg-save">
```

Comment:
please change the icon to 24x24
it can be found [here](https://www.figma.com/file/LXl9BAJSwtgWuzkxqOqvX8/ML1-Design-System?node-id=53015%3A36793&t=SAqJtQPlTsvKXOIb-0)